README_HW6.md

```diff
@@ -0,0 +1,27 @@
+## Part 1 - Short answer questions
+
+a) In SwiftUI, you describe your interfaces **declaratively** and leave the **rendering** to the Framework.
+
+b) When a View first appears, and you want it to animate in, you can do so in the Views **`.onAppear`** modifier.
+
+c) In Swift, the native types **Int** and **String** implement the Hashable protocol
+
+d) **Source** is how long an animation takes to complete.
```

Comment:
Duration not Sources

Reply:
Don't know where "Sources" came from—I don't think I meant to type that!

Reply:
i figured